backend/fms_core/utils.py

```diff
@@ -73,6 +73,13 @@ def normalize_scientific_name(name: str) -> str:
     """
     return " ".join((a.title() if i == 0 else a.lower()) for i, a in enumerate(RE_WHITESPACE.split(name or "")))
 
+def str_remove_parenthesis(s: str | None) -> str | None:
```

Comment:
This is not a generic function you should put that function in the importer file where it is used ... i doubt we'll reuse this ...